.github/workflows/ci.yml

```
@@ -54,10 +54,11 @@ jobs:
       - name: Evaluate definitions
         id: definitions
         run: |
-          export MSRV=$(rustup show | awk 'NF' | awk 'END{print $2}')
+          rustup show active-toolchain
+          export MSRV=$(rustup show active-toolchain | awk -F'-' '{print $1}')
```

I'd prefer to check that this actually returns a non-empty value in case it breaks in the future

```suggestion
          export MSRV=$(rustup show active-toolchain | awk -F'-' '{print $1}')
          if [ -z "$MSRV" ]; then
            echo "Error: MSRV did not parse correctly"
            exit 1
          fi
```